plugins/xpad/xpad.cpp

```diff
@@ -23,6 +23,10 @@
 #include "xpad.h"
 #include <VersionHelpers.h>
 
+#undef _WIN32_WINNT
+#define _WIN32_WINNT 0x0602 // Required for XInputEnable definition
```

Comment:
...What. I have no clue what you are talking about, or what you're worried about.